src/screens/MainApp/DiagnoseRequest/screens/Gallery/index.js

```diff
@@ -0,0 +1,197 @@
+import React, { useState, useEffect, useCallback, useMemo } from 'react'
+import CameraRoll from '@react-native-community/cameraroll'
+import { FlatList, View, Alert, PermissionsAndroid } from 'react-native'
+import { SvgXml } from 'react-native-svg'
+import Background from '~/components/Background'
+import { PrimaryButton, Description, Button, Body } from '~/components'
```

Comment:
I would change this to more specific imports

`import { PrimaryButton, Button } from '~/components/buttons'`
`import { Description, Body } from '~/components/texts'`